src/coreclr/vm/jithelpers.cpp

```diff
@@ -2368,26 +2352,34 @@ HCIMPLEND
 // Currently, counter is a pointer into the Tier0 method stack
 // frame so we have exclusive access.
 
-void JIT_Patchpoint(int* counter, int ilOffset)
+extern "C" void JIT_PatchpointWorkerWorkerWithPolicy(TransitionBlock * pTransitionBlock)
```

Comment:
Comment above should be updated to reflect that this helper has different policies for OSR patchpoints and partial compilation patchpoints.